inc/search.class.php

```diff
@@ -4186,38 +4356,39 @@ static function addWhere($link, $nott, $itemtype, $ID, $searchtype, $val, $meta
                 && ($val != 'NULL') && ($val != 'null')) {
                $toadd2 = " OR `$table`.`$field` IS NULL";
             }
-            return $link." (((`$table`.`$name1` $SEARCH
-                            $tmplink `$table`.`$name2` $SEARCH
-                            $tmplink `$table`.`$field` $SEARCH
-                            $tmplink CONCAT(`$table`.`$name1`, ' ', `$table`.`$name2`) $SEARCH )
+            return $link." (((".$this->db->quoteName("$table.$name1")." ".$this->getSearchFromType($searchtype, $val, $nott, $inittable, true)."
```

Comment:
Maybe `$this->getSearchFromType($searchtype, $val, $nott, $inittable, true)` could be placed in a variable, instead of being called 4 times in same string concatenation.